### DISCUSSION TITLE
Lapis Itemdb

### DISCUSSION THREAD

**Discussion by SophieFox23** [EssentialsX/Essentials#4951] [Ideas and Feature Suggestions]
Hey, an idea is to add a shortened itemdb for lapislazuli down to maybe lapis as the full name takes up alot of room on the sign for [trade] and maxes the amount you can store to 9
![2022-06-16_01 32 58](https://user-images.githubusercontent.com/83872862/173998469-15781340-7349-4c5c-8b17-e9a38cedc67f.png)

**Comment by SophieFox23** [comment]
Other ones to add to get a shorter name would be:

Dark oak Planks
Dark oak Saplings

**Comment by triagonal** [comment]
There definitely are a few items that could use some shorter names for trade signs - for now, you can use the `custom_items.yml` file in your Essentials folder to define shorter aliases. The file contains a few examples you can use to get started.

**Comment by oskarkk** [comment]
The alias system is kinda broken, see my issue: #4861 (fun fact: there are literally 10000 different aliases for all the copper blocks). It'd be nice if it was somehow reworked.
